src/App/Router.php

```diff
@@ -40,7 +40,42 @@ class Router
 	 */
 	public function collectRoutes()
 	{
-		$this->routeCollector->addRoute(['GET', 'POST'], '/itemsource[/{guid}]', Module\Itemsource::class);
+		$this->routeCollector->addGroup('/admin', function (RouteCollector $collector) {

```

Comment:
Yes, it will contain the explicit and exhaustive list of all the routes we're serving.

For `mod/display`, this gives something like:
```php
$this->routeCollector->addRoute(['GET'], '/display/{guid:[a-z0-9-]+}'                   , Module\Item\Display::class);
$this->routeCollector->addRoute(['GET'], '/display/{nick}/{id:\d+}'                     , Module\Item\Display::class);
$this->routeCollector->addRoute(['GET'], '/display/feed-item/{id:\d+}.atom'             , Module\Item\Display::class);
$this->routeCollector->addRoute(['GET'], '/display/feed-item/{id:\d+}/conversation.atom', Module\Item\Display::class);
```

for example. I had to dig into the module to find the last two out. Now it's going to be explicit.

Reply:
What's the problem with dynamic functions?

Still, it's a good idea by itself.

Reply:
The more I think about it, the more I think we should have a separate `static/` folder with all the static data of the app, including the db structure, the config defaults and the routes. This way the config folder would be cleared of non-user files. This would make the update process as easy as it was before I started to put a lot of non-user files in it.

Reply:
Some additional question: Do we now have the possibility to define in ```addRoute``` that it is a backend routine, so that we don't initiate a session and so on. Currently this is done via a separate list of endpoints in App.php.

Reply:
This distinction should be done at the module level via a class variable. First we do the routing, then from the collected Module class we know if it's a backend routine or not.

Reply:
Okay. I only wanted to make sure that we don't forget about it.